src/main/java/Duke.java

```diff
@@ -1,10 +1,176 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+    //define constants
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_DONE = "done";
+    private static final String COMMAND_BYE = "bye";
+
+    private static final String DEADLINE_BY_PREFIX = "/by ";
+    private static final String EVENT_AT_PREFIX = "/at ";
+
+    private static Task[] tasks = new Task[100];
+
+    /*
+    Standard text output by pepepopo
+     */
+    public static void greeting() {
+        String greeting = " Hello! I'm Pepepopo\n" +
+                        " What can I do for you?";
+        System.out.println(greeting);
+        divider();
+    }
+
+    public static void divider() {
```

Comment:
Same comment as above. Perhaps this could use a naming as "printDivider" instead? :)